pyEpiabm/pyEpiabm/utility/inverse_cdf.py

```diff
@@ -0,0 +1,72 @@
+#
+# Class of inverse CDF as is done in Covidsim code.
+#
+
+import random
+import pyEpiabm as pe
+import math
+import numpy as np
+
+
+class InverseCdf:
+    """Class of inverse cumulative distribution functions and their associated
+    methods, in a style similar to CovidSim.
+    """
+
+    def __init__(self, mean, icdf_array):
+        """Constructor Method
+
+        :param mean: Mean of the icdf
+        :type mean: float
+        :param icdf_array: Array of quantiles of the icdf_array, values
+        in array must be approximately evenely spaced
+        :type icdf_array: numpy array
+        :param CDF_RES: Resolution of icdf taken as length of icdf array
+        :type: int
+        :param time_steps_per_day: number of time steps per day in simulation
+        :type: int
```

Comment:
```suggestion
```
Good to document these, but having them here implies they are arguments of the function - maybe just comment in main body or at the top of the docstring?